contracts/GenArt721CoreV3.sol

```diff
@@ -963,7 +1025,31 @@ contract GenArt721CoreV3 is
         external
         onlyUnlocked(_projectId)
         onlyArtistOrAdminACL(_projectId, this.updateProjectAspectRatio.selector)
+        onlyNonEmptyString(_aspectRatio)
```

Comment:
nit: should we also be using `onlyValidProjectId` since adminACL can call this?

Reply:
See: https://github.com/ArtBlocks/artblocks-contracts/pull/310#discussion_r981381481